public_dropin_gpu_environments/nemollm_inference_ms/Dockerfile

```diff
@@ -78,6 +78,11 @@ ENV PIP_NO_CACHE_DIR=1
 ENV PATH=${BASE_PATH}
 RUN python -m venv $DATAROBOT_VENV_PATH && $DATAROBOT_VENV_PATH/bin/pip install -U pip
 COPY dr_requirements.txt dr_requirements.txt
+
+# artifact downloader tool
+COPY artifact_downloader/ ./artifact_downloader/
+ENV PYTHONPATH=$PYTHONPATH:${CODE_DIR}/artifact_downloader/
```

Comment:
I moved the ModelDownloader code to a separate PR:
1) the code change is big
2) I'm not the original owner of this code, but we want to maintain and extend it
https://github.com/datarobot/datarobot-user-models/pull/965/files

Reply:
I decided to remove the ModelDownloader from this release, until we fix all the issues related to it.

Reply:
will remove those lines

Reply:
removed.